### ISSUE TITLE
Customization Lighting Missing a (spot?-)light

### ISSUE THREAD

**Issue by CyberDNIWE** [TekkenOverlay/TekkenOverlay#148]
In character customization a there is a noticable absence of light on the head of the character. I suspect a spotlight source is either missing or is facing an incorrect direction. Below is a comparison with left being non-injected version and right - the injected one.
All settings are left at the defaults and no camera movements were made after entering preset customization room.
![Left-vanilla--right-injected](https://i.imgur.com/xqFV1jz.png)

The `Tekken8OverlaySettings.yaml` is included as a `.txt` because github doesnt allow for its embedding.
[Tekken8OverlaySettings.yaml](https://github.com/TekkenOverlay/TekkenOverlay/files/14078472/Tekken8OverlaySettings.txt)

**Comment by thecoolestlol** [comment]
Yes, just like Sanctum is almost pitch black, especially when you break the wall. This game was not made for us to disable the lighting and force lumen and such.

One solution I find is to use custom character lighting and make it very close to black so it is subtle, but allows you to see the characters within environments that are too dark.

Sadly those settings get reset every time you go into a match so it's pointless to use unless you know how to set up the custom commands to make it work on its own, which I don't.

**Comment by CyberDNIWE** [comment]
> Yes, just like Sanctum is almost pitch black, especially when you break the wall. This game was not made for us to **disable the lighting** and force **lumen** and such.

My problem is that the overlay messes up the lighting **by default.** I just don't get why. It's a cool addition, sure, but I would prefer if it didn't intervene with anything, unless explicitly instructed to.
I personally use it only for unlocking the fps, which I think is the most frequent use-case for the tool.
